src/System.Net.Sockets/src/System/Net/Sockets/SafeSocketHandle.Windows.cs

```diff
@@ -229,6 +229,9 @@ internal static InnerSafeCloseSocket Accept(SafeSocketHandle socketHandle, byte[
                 }
                 return result;
             }
+
+            internal void TryUnblockSocket(bool abortive)
+            { }
```

Comment:
Should we be using CancelIoEx here?

Reply:
Yes, that was one part missing in original PR.